ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -134,7 +135,11 @@ export const styles = reactStyles({
     display: 'flex',
     flexDirection: 'column',
     alignItems: 'center',
-    height: '10rem'
+    height: '10rem',
+    marginTop: '2rem',
```

Comment:
Moving these styles in with this style class, because it is only used in one place.